math/statistics.js

```diff
@@ -4,12 +4,89 @@ define(() => {
 	// sort() defaults to string sort
 	const numericSort = (a, b) => (a - b);
 
+	const numericSortTagged = (a, b) => (a.val - b.val);
+
+	//Used to compute the significance of the Mann-Whitney U test
+	//Since we are comparing adjacently ranked entries, ties are sure to
+	//abound. Therefore, the approximate distribution is used to avoid the
+	//computationally intensive task of exhaustively enumerating the sampling
+	//distribution of U.
+	const mwMean = (a, b) => a*b/2;
```

Comment:
This should probably be written with the function keyword to match the style of other helper functions in this file. (because `mwMean` is never treated like a variable; it's only ever called as a function).